### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: examples/02-HFSS/Advanced_Far_Field.py [hunk 1]
```diff
@@ -183,9 +183,14 @@ def ff_calc(x=0, y=0, qty="rETotal", dB=True):
     setup_sweep_name=hfss.nominal_sweep, expression="RealizedGainTotal", domain="Elevation"
 )
 
-
+###############################################################################
+#  Polar Plot
+# ~~~~~~~~~~~
```
Comment (akaszynski) [inline examples/02-HFSS/Advanced_Far_Field.py:188]:
```suggestion
# Polar Plot
# ~~~~~~~~~~
```

File: examples/02-HFSS/HFSS_Dipole.py [hunk 2]
```diff
@@ -117,6 +117,17 @@
     report_category="Far Fields",
 )
 
+###############################################################################
+# Postprocessing
+# --------------
+# Create post processing variable and assign to new coordinate system .
```
Comment (akaszynski) [inline examples/02-HFSS/HFSS_Dipole.py:123]:
```suggestion
# Create post processing variable and assign to new coordinate system.
```

File: examples/05-Q3D/Q2D_Example_CPWG.py [hunk 3]
```diff
@@ -129,25 +134,30 @@
...
+
 obj = q.modeler.get_object_from_name("signal")
 q.assign_single_conductor(
     name=obj.name, target_objects=[obj], conductor_type="SignalLine", solve_option="SolveOnBoundary", unit="mm"
 )
-# ~~~~~~~~~~~~~~~~~
+
+###############################################################################
 # Reference ground
+# ~~~~~~~~~~~~~~~~~
```
Comment (akaszynski) [inline examples/05-Q3D/Q2D_Example_CPWG.py:145]:
```suggestion
# ~~~~~~~~~~~~~~~~
```

File: examples/06-Multiphysics/Hfss_Icepak_Coupling.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 """
-HFSS-Icepack Multyphisics Analysis
-----------------------------------
+Multiphisics: HFSS-Icepack Multyphisics Analysis
+-------------------------------------------------
```
Comment (akaszynski) [inline examples/06-Multiphysics/Hfss_Icepak_Coupling.py:3]:
```suggestion
------------------------------------------------
```

**Review by akaszynski — APPROVED** [review]
LGTM with suggestions.

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**